src/PaymentSuite/RedsysBundle/Tests/Functional/Services/RedsysFormTypeWrapperTest.php

```diff
@@ -155,11 +159,11 @@ public function setUp()
      */
     public function testFormCreation()
     {
-        $amount = 10;
+        $amount = 100;
 
         $formData = array(
-            'Ds_Merchant_Amount'             => $amount * 100,
-            'Ds_Merchant_MerchantSignature'  => '11473E3E726C16798532E9924444953A5C12DB2C',
+            'Ds_Merchant_Amount'             => $amount,
+            'Ds_Merchant_MerchantSignature'  => 'CB43B12351A9826D9640CC285CBDFD8CA6A5994C',
```

Comment:
can you explain me what is this identifier ? it is from Redsys platform ? do we need a real identifier ?